packages/cosmic-swingset/src/block-manager.js

```diff
@@ -79,6 +80,17 @@ export default function makeBlockManager({
         break;
       }
 
+      case ActionType.INSTALL_BUNDLE: {
```

Comment:
BTW my note about "commit swingstore state" on the `solo` side is not a concern here, the block-manager is already in a context where the state will be committed once block processing is complete